### PR TITLE
fix quarkus patch, downgrade quarkus platform

### PR DESCRIPTION
**Pull request by jrangelramos**
* Update RH GA quarkus platform.

### CODE REVIEW

**Comment by jrangelramos** [comment]
The current platform version set `2.13.5.Final-redhat-00003`  does not exists on Red Hat maven repo.

```
$ curl -s -o /dev/null -w "%{http_code}\n" https://maven.repository.redhat.com/ga/com/redhat/quarkus/platform/quarkus-bom/2.13.5.Final-redhat-00003/
404
$ curl -s -o /dev/null -w "%{http_code}\n" https://maven.repository.redhat.com/ga/com/redhat/quarkus/platform/quarkus-bom/2.13.5.Final-redhat-00002/
200
```

File: openshift/patches/0003-quarkus-productized.patch [hunk 1]
```diff
@@ -58,7 +58,7 @@ index 2bc2df55..e18ac0f3 100644
 -    <quarkus.platform.group-id>io.quarkus.platform</quarkus.platform.group-id>
 -    <quarkus.platform.version>2.16.0.Final</quarkus.platform.version>
 +    <quarkus.platform.group-id>com.redhat.quarkus.platform</quarkus.platform.group-id>
-+    <quarkus.platform.version>2.13.5.Final-redhat-00003</quarkus.platform.version>
++    <quarkus.platform.version>2.13.5.Final-redhat-00002</quarkus.platform.version>
```
Comment (matejvasek) [inline openshift/patches/0003-quarkus-productized.patch:61]: This change should be done twice -- for both CE and HTTP template.

**Comment by matejvasek** [comment]
I used:
```
curl -s "https://code.quarkus.redhat.com/api/platforms" | jq -r '.platforms[0].streams[0].releases[0]'
```
to determine version.
This is odd.

File: openshift/patches/0003-quarkus-productized.patch (hunk 1 above)
Comment (jrangelramos) [reply openshift/patches/0003-quarkus-productized.patch:61]: nice catch..

**Comment by jrangelramos** [comment]
yea.. now that api points to `-0002`. They might be rolled back..

**Comment by jrangelramos** [comment]
Closing in favor of https://github.com/openshift-knative/kn-plugin-func/pull/469

**Pull request closed without merging**